core/templates/dev/head/pages/exploration_editor/editor_tab/TrainingModalService.js

```diff
@@ -0,0 +1,108 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Controllers for the graphical state editor.
+ */
```

Comment:
Also here, please add a newline after this.

Reply:
@shubha1593  I have added the new line and I have made the TrainingDataServiceSpec.js

Reply:
@shubha1593  Can u please help me with "Resolve Confict" , I have just cut from StateEditorSpec.js and pasted it in TrainingDataServiceSpec.js

Reply:
Here too, the fileoverview comment will get updated! You can again do this in the same manner bringing the comment below over here. So it will be like `@fileoverview Service which handles .....`

Reply:
@shubha1593 I have made changes to the comment as you said.